src/quo2/components/profile/select_profile/view.cljs

```diff
@@ -0,0 +1,53 @@
+(ns quo2.components.profile.select-profile.view
+  (:require
+    [quo2.components.profile.select-profile.style :as style]
+    [react-native.core :as rn]
+    [quo2.components.markdown.text :as text]
+    [quo2.components.avatars.user-avatar :as user-avatar]
+    [reagent.core :as reagent]))
+
+(defn- on-change-handler
+  [selected? on-change]
+  (swap! selected? not)
+  (when on-change (on-change @selected?)))
+
+(defn view
+  "Options
+   - `default-selected?` Default selected state
+   - `selected?` Selected state
```

Comment:
Hi @ajayesivan, Thank you for PR. Why we need two states here?

Reply:
`default-selected?` -  provide the initial state when using it as an uncontrolled component.
`selected?` - use when the component is controlled.

We are using similar patterns in quo2 selector components.

I have updated the doc string with this info.